scripts/check-logids.js

```diff
@@ -0,0 +1,75 @@
+const { runInDir } = require('./run-in-dir');
+
+// Gather all log ids from the Compass source,
+async function main() {
+  const { stdout } = await runInDir(`git grep -H mongoLogId`);
+  const mapNumberToSource = new Map();
+
+  for (const line of stdout.split('\n').filter(Boolean)) {
+    // git grep -H gives us `${filename}:${source}`
+    const { filename, source } = line.match(/^(?<filename>[^:]+):(?<source>.+)$/).groups;
+    if (filename.match(/\.spec\.(js|jsx|ts|tsx)$/))
```

Comment:
There are a few tests files with `.test.js` in some of the older packages, do we want to also add a skip for those files here to?

Reply:
TIL :)

Done! Generally speaking, I think it’s just fine if we adjust the script as changes happen as well :)